tools/cldr-code/src/test/resources/org/unicode/cldr/unittest/data/common/dtd/ldml.dtd

```diff
@@ -1,1331 +1,3288 @@
 <!--
-Copyright © 2003-2010 Unicode, Inc. and others. All rights reserved. Distributed under the Terms of Use in http://www.unicode.org/copyright.html.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of the Unicode data files and any associated documentation (the "Data Files") or Unicode software and any associated documentation (the "Software") to deal in the Data Files or Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, and/or sell copies of the Data Files or Software, and to permit persons to whom the Data Files or Software are furnished to do so, provided that (a) the above copyright notice(s) and this permission notice appear with all copies of the Data Files or Software, (b) both the above copyright notice(s) and this permission notice appear in associated documentation, and (c) there is clear notice in each modified Data File or in the Software as well as in the documentation associated with the Data File(s) or Software that the data or software has been modified.
+Copyright © 1991-2023 Unicode, Inc.
```

Comment:
this whole file is test data, right? Why so many changes?

Reply:
does this match another ldml.dtd elsewhere?

Reply:
It's an exact update of the production ldml.dtd.

it's frozen here for testing purposes.

Reply:
`- update the 'testdata' common tree, with a new DTD`

Reply:
@btangmu it also shows how we can work with a very controlled data tree, might be needed for inheritance tests.